ffw/UIRPC.js

```diff
@@ -199,8 +199,29 @@ FFW.UI = FFW.RPCObserver.create(
           case 'UI.SubtleAlert':
           {
             if (SDL.SDLModel.onUISubtleAlert(request.params, request.id)) {
-              SDL.SDLController.onSystemContextChange(request.params.appID);
+              if(!request.params.softButtons){
+                SDL.ResetTimeoutPopUp.extendResetTimeoutRPCs([request.method]);
+                SDL.ResetTimeoutPopUp.extendResetTimeoutCallBack(SDL.SubtleAlertPopUp.setTimerUI , request.method);
+                SDL.ResetTimeoutPopUp.setContext(request.method);
+                SDL.ResetTimeoutPopUp.expandCallbacks(function(){
+                  SDL.SubtleAlertPopUp.deactivate('timeout');
+                }, request.method);
+
+                if (request.params.duration) {
+                  SDL.ResetTimeoutPopUp.set('timeoutSeconds',
+                    {
+                      'UI.SubtleAlert': request.params.duration/1000,
+                      'TTS.Speak': request.params.duration/1000
```

Comment:
@VadymLuchko duration/1000 - is it convertation to seconds? Move hardcoded number to logical named const

Reply:
fixed in 9ad1a68